services/drupal/web/modules/custom/epa_core/epa_core.module

```diff
@@ -373,13 +378,43 @@ function epa_core_tokens($type, $tokens, array $data, array $options, Bubbleable
           $term_description = $token->replace($term_description, ['term' => $data['term']]);
           $replacements[$original] = $term_description;
         }
-
         break;
     }
   }
   return $replacements;
 }
 
+function epa_core_addtocal_alter(array &$overridable_items) {
+  $overridable_items['override_status'] = TRUE;
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node->bundle() == 'event') {
+    $cal_description = $node->get('field_add_to_cal_description')->value;
+    $default_description = $node->get('field_description')->value;
+    if (!empty($cal_description)) {
+      $description = $cal_description;
+    } elseif (!empty($default_description)) {
+      $description = $default_description;
+    }
+    $event_type = Term::load($node->get('field_event_type')->target_id)->getName();
```

Comment:
You don't need to go back and change this, but I believe you could just do: `$node->field_event_type->entity->name` @pzh7000